WallTrailingGame/src/main/java/com/projecttango/examples/java/planefitting/ConfigActivity.java

```diff
@@ -56,12 +60,46 @@ protected void onCreate(Bundle saveIntentState) {
         super.onCreate(saveIntentState);
         setContentView(R.layout.activity_config);
 
+        array.add("Wall Distance (feet)");
+        array.add("1.0");
+        array.add("1.5");
+        array.add("2.0");
+        array.add("2.5");
+        array.add("3.0");
+        array.add("3.5");
+        array.add("4.0");
+
+
         //Initializing variable mSelectSpinner
         mSelectSpinner = (Spinner) findViewById(R.id.spinnerDistance);
+        arrayAdapter = new ArrayAdapter<String>(this, R.layout.spinner_item, array);
+        mSelectSpinner.setAdapter(arrayAdapter);
         mSelectSpinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
             public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
                 Object item = parent.getItemAtPosition(position);
-                mRewardSoundDist = Double.parseDouble(item.toString());
+                if(mRewardSoundDist != 0.0){
+                    if(!item.toString().equals("Wall Distance (feet)")){
+                        mRewardSoundDist = Double.parseDouble(item.toString());
+                    }
+                }
+                else{
+                    if(mediaPlayer == null && !item.toString().equals("Wall Distance (feet)"))
+                    {
+                        mButton.setText("Play");
```

Comment:
Mind also replacing all instances of mButton to startPauseButton?  It'll make it a lot more clear when we look at the code weeks later.